City-of-Helsinki/helsinki-paatokset
# Pp 0 develop fixes 2022 01 18

Fix webpack + alter configs to Helsinki Kanava integration

To  test:
- Checkout branch. Run `drush cr; drush cim -y`
- This should get rid of meeting announcement on the front page and `/kaupunginvaltuusto` page UNLESS you have a meeting scheduled to start
- Try running `drush mim paatokset_meeting_videos` migration. You should see the most recent meeting record on `/kaupunginvaltuusto` page. The link 'all videos' link below the video should work.
- Go to `public/themes/custom/helfi_paatokset`. Try running `npm run dev` and `npm run build` commands. They should work.
- Click around the site. Styles should be built correctly. If you open the network tab and filter for .js files, all of them should load correctly.
